finagle-redis/src/main/scala/com/twitter/finagle/redis/Client.scala

```diff
@@ -82,19 +66,46 @@ class BaseClient(service: Service[Command, Reply]) {
     }
 
   /**
-   * Closes connection to Redis instance
+   * Select DB with specified zero-based index
+   * @param index
    */
-  def quit(): Future[Unit] =
-    doRequest(Quit) {
+  def select(index: Int): Future[Unit] =
```

Comment:
They were in BaseClient. I moved them to Client, because they are not available in sentinel and/or cluster mode.